draft-wood-tls-external-psk-importer.md

```diff
@@ -88,9 +90,10 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 # Key Import
 
 A key importer takes as input an EPSK with external identity 'external_identity' and base key 'epsk',
-as defined in {{terminology}}, along with an optional label, and transforms it into a set of PSKs and
-imported identities for use in a connection based on supported HashAlgorithms. In particular, for each
-supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity structure as follows:
+as defined in {{terminology}}, along with an optional label, and transforms
+it into a set of PSKs and imported identities for use in a connection based on supported HashAlgorithms.
+In particular, for each supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity
+structure as follows:
```

Comment:
(I didn't find any changed text here. Just to confirm, this was just a reflow?)

Reply:
Yep!